processor/metricstransformprocessor/metrics_transform_processor_testcases_test.go

```diff
@@ -411,18 +473,25 @@ var (
 				},
 			},
 			in: []*metricspb.Metric{
-				metricBuilder().setName("metric1").setLabels([]string{"label1", "label2"}).setDataType(metricspb.MetricDescriptor_GAUGE_INT64).
-					addTimeseries(3, []string{"label1-value1", "label2-value1"}).addInt64Point(0, 3, 2).
-					addTimeseries(0, []string{"label1-value1", "label2-value4"}).
```

Comment:
This test is important to retain since timestamp "0" is used to test the nil start timestamp case.

Reply:
done.